src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
+				reply = list.join ' '
+				context.send reply
```

Comment:
I think it would be nice to echo the existing message with the pings included, that way I would see the context of the ping in my notification directly.  That's particularly useful on my phone, so I can see what the ping is about at a glance without having to load the entire flowdock app/thread and therefore cancelling the notification (when I might not be able to deal with it at the time)